drivers/sensor/adxl362/Kconfig

```diff
@@ -77,6 +77,19 @@ config ADXL362_TRIGGER_OWN_THREAD
 
 endchoice
 
+config ADXL362_WAKEUP_MODE
+	bool "Wake-up mode"
+	help
+	  This mode is intended for extremely low power consumption,
+	  checking for motion only about six times a second.
+
+config ADXL362_AUTOSLEEP
+	bool "Autosleep feature"
+	help
+	  Enter Wake-Up mode when inactivity is detected,
+	  reenter Measurement mode when activity is detected.
+	  Only applies for Linked and Loop mode, ignored otherwise.
```

Comment:
please make this a Devicetree property, so that it can be set per-instance. e.g. `power-mode`.